<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#centralwidget{
	background-image:url(img/mainwindow.jpg);
}
QGroupBox::title {
	color:#ff0000
}

QTableWidget
{
background: rgb(237,216,255);
alternate-background-color:rgb(48,51,55);
selection-background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(237,216,255),stop:1 rgb(237,211,255));
}

QHeaderView::section
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, 		stop:0 rgb(237,216,255),
	stop:1 rgb(237,211,255));
color: rgb(0,0,0);
padding-left: 4px;
border: 1px solid #383838;
}
QScrollBar:vertical 
{
border: 0px solid grey;
background: transparent;
width: 15px;
margin: 22px 0 22px 0;
}
QScrollBar::handle:vertical 
{
background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 rgb(46,46,46),stop:1 rgb(66,66,66));
min-height: 20px;
}
QScrollBar::add-line:vertical 
{
border: 0px solid grey;
background: rgb(66,66,66);
height: 20px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical 
{
border: 0px solid grey;
background: rgb(56,56,56);
height: 20px;
subcontrol-position: top;
subcontrol-origin: margin;
}
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical 
{
border: 0px solid grey;
width: 3px;
height: 3px;
background: rgb(46,46,46);
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
background: none;
}

QPushButton{
color:#0000ff;
border:26px;
solid #FFFFFF;
border: 1px solid green;
border-radius:10px;
}
QPushButton:hover{
color:#FF0000;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>基本请求</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox::title{
font: 75 14pt &quot;Agency FB&quot;;
}</string>
      </property>
      <property name="title">
       <string>获取基本信息</string>
      </property>
      <widget class="QPushButton" name="btn_AddDevice">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>16</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>添加新设备</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>221</width>
         <height>122</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ECUID：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>版本号：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>信号强度：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>RSD类型：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>软件版本号：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_ECUID"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_TYPE">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_SIGNAL_LEVEL">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_Equipment_Status">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_Version">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>信道：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_SIGNAL_CHANNEL">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>193</y>
        <width>241</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>信道设置</string>
      </property>
      <widget class="QComboBox" name="comboBox_SetChannel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>21</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>02</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>03</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>04</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>05</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>06</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>07</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>08</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>09</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>新信道：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_SetChannel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>21</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>设置信道</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>当前信道：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>信号强度：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_CurrentChannel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_Stren">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>319</y>
        <width>241</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>设置WIFI密码</string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>原密码：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>新密码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_oldPasswd">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_newPasswd">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_setPasswd">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>设置密码</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>511</width>
        <height>411</height>
       </rect>
      </property>
      <property name="title">
       <string>设置组网</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_ID">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_addID">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>添加ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_SetID">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>设置组网</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_ID">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>191</width>
         <height>351</height>
        </rect>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>需要添加的ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>需要注册的ID列表</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_ECUImport">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ECU导入ID</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>系统信息</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string>系统信息</string>
      </property>
      <widget class="QPushButton" name="btn_getSystem">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>75</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>获取系统信息</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>41</y>
         <width>761</width>
         <height>371</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>设备状态</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>开关机</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>功能开关</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PV1欠压</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PV1欠压</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>心跳次数</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>超时次数</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>当天关机数</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PV1输入电压</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PV2输入电压</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>输入电流</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PV1输入功率</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PV2输入功率</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
